ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -755,39 +766,33 @@ define([
       }
     },
 
-    // Convert Lat-Lon to UTM coordinates. Returns undefined if lat or lon is undefined or not a number.
+    // Convert Lat-Lon to UTM/UPS coordinates. Returns undefined if lat or lon is undefined or not a number.
     // Returns undefined if the underlying call to usng fails. Otherwise, returns an object with:
     //
     //   easting    : FLOAT
     //   northing   : FLOAT
-    //   zoneNumber : INTEGER (>=1 and <= 60)
+    //   zoneNumber : INTEGER (>=0 and <= 60)
     //   hemisphere : STRING (NORTHERN or SOUTHERN)
-    LLtoUTM: function(lat, lon) {
+    LLtoUtmUps: function(lat, lon) {
       if (isNaN(lat) || isNaN(lon)) {
         return undefined
       }
-
-      var utmCoords = []
-      var converterReturnCode = converter.LLtoUTM(lat, lon, utmCoords)
-
-      if (converterReturnCode === 'undefined') {
-        return undefined
-      }
-
-      var results = {}
-      results.easting = utmCoords[0]
-      results.northing = lat >= 0 ? utmCoords[1] : utmCoords[1] + northingOffset
-      results.zoneNumber = utmCoords[2]
-      results.hemisphere = lat >= 0 ? 'NORTHERN' : 'SOUTHERN'
-
-      return results
+      const utmUps = converter.LLtoUTMUPSObject(lat, lon)
```

Comment:
I thought we were taking all this logic out of location because we can use the new functions here that do it for us: https://github.com/codice/usng.js/commit/5a0d1465aed6b86e07681642fdaa877a012e9623

Hopefully the LLtoUTMUPSObject are already using these functions for the UTM side and the UTM comes back correct

Reply:
We pushed that back as a stretch goal since we're already seeing issues with drawing when the UTM UPS tab is selected.